src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestHelpers/TestRunner.cs

```diff
@@ -309,6 +309,10 @@ internal static void TestToString(FormulaValue result, StringBuilder sb)
                 // $$$ proper escaping?
                 sb.Append('"' + s.Value + '"');
             }
+            else if (result is GuidValue guid)
+            {
+                sb.Append(guid.Value);
+            }
```

Comment:
Nit: can we have this emit a fx expression, 
Guid("xxxx") 

like we do for date on line 392. 
Helps keep everything unambiguous.